docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -411,15 +301,100 @@ Payload:
 }
 ```
 
-## Skip organization role sync
+## Examples of setting up generic OAuth2
 
-To prevent the sync of organization roles from the OAuth provider, set `skip_org_role_sync` to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.
-This also impacts the `allow_assign_grafana_admin` setting by not syncing the Grafana admin role from the OAuth provider.
+### Set up OAuth2 with Auth0
```

Comment:
Is this a new task that is unrelated to the Configure task? We need an introduction that helps explain what this is and why a user might want to complete this task. Are there any before you begin items to mention?